source/tutorials/quality-control.rst

```diff
@@ -0,0 +1,45 @@
+Removing contaminant sequence data with q2-quality-control
+==========================================================
+
+.. note:: This guide assumes you have installed QIIME 2 using one of the procedures in the :doc:`install documents <../install/index>` and completed the :doc:`moving pictures tutorial <moving-pictures>`.
+
+This tutorial will demonstrate how to use the ``q2-quality-control`` plugin to filter sequence data.
+
+We will download and create several files, so first create a working directory.
+
+.. command-block::
+   :no-exec:
+
+   mkdir quality-control-tutorial
+   cd quality-control-tutorial
+
+Excluding sequences by alignment
+--------------------------------
+
+The `exclude-seqs` method aligns a series of query sequences contained in a ``FeatureData[Sequences]`` file against a series of reference sequences. This method employs a number of different alignment criteria (BLAST evalue, percent identity to top reference sequence, and percent of query that aligns to top reference sequence) to decide whether that sequence "hits" the reference sequences, and outputs separate files of sequences that hit/do not hit the reference sequences. This method can be used for a variety of applications, including removing known contaminant sequences, excluding host sequences (e.g., human DNA), or removing non-target sequences (e.g., non-bacterial) from your data.
```

Comment:
Use double backticks to have `exclude-seqs` render correctly (it's currently italicized).

Reply:
Change `FeatureData[Sequences]` to `FeatureData[Sequence]` (we don't have the plural form defined yet).

Reply:
You mention "series of query sequences" and "series of reference sequences". The word "series" implies order -- does the order matter here? If not, perhaps "set of query sequences" and "set of reference sequences" would be clearer.

Reply:
👍